Sources/PackageModel/Target/SwiftTarget.swift

```diff
@@ -129,14 +129,14 @@ public final class SwiftTarget: Target {
 
     override public func encode(to encoder: Encoder) throws {
         var container = encoder.container(keyedBy: CodingKeys.self)
-        try container.encode(self.swiftVersion, forKey: .swiftVersion)
+        try container.encode(self.toolSwiftVersion, forKey: .swiftVersion)
```

Comment:
@MaxDesiatov is it okay to rename `case` in `CodingKeys` here? I'm not sure how much do we care about backward compatibility here.

Reply:
I'm still unsure where this `Codable` conformance is used, maybe @neonichu could clarify? I wonder if this could be used in workspace serialization, but also don't know how easily we can break that

Reply:
I can do this in a follow up!